docs/stdlib/bytes.rst

```diff
@@ -123,10 +129,10 @@ Bytes
 
 .. eql:function:: std::bytes_get_bit(bytes: bytes, nth: int64) -> int64
 
-    Get the *nth* bit of the *bytes* value.
+    Returns the ``nth`` bit of ``bytes`` as a value of :eql:type:`int64`.
```

Comment:
Move references from one-liner to details and style them with emphasis (`*`) instead of as inline code.